src/modules/email/mentionEmail.js

```diff
@@ -2,68 +2,49 @@
 /* eslint quotes: 0*/
 import { config } from '../../core-server';
 import * as Constants from '../../lib/Constants';
-import Counter from '../../lib/counter';
 import Logger from '../../lib/logger';
 import fs from 'fs';
 import jwt from 'jsonwebtoken';
 import * as pg from '../../lib/pg';
 import send from './sendEmail';
+import promisify from '../../lib/promisify';
 import handlebars from 'handlebars';
-import getMailObj from './buildMailObj';
 const MENTION_INTERVAL = 60 * 60 * 1000, MENTION_DELAY = 60 * 60 * 1000,
 	template = handlebars.compile(fs.readFileSync(__dirname + '/../../../templates/' +
-	config.app_id + '.digest.hbs', 'utf-8').toString()), log = new Logger(__filename),
-	connStr = config.connStr, conf = config.email, counter = new Counter();
+	config.app_id + '.mention.hbs', 'utf-8').toString()), log = new Logger(__filename, 'mention'),
+	connStr = config.connStr, conf = config.email;
 
 let lastEmailSent, end, i = 0;
 
 function initMailSending (userRel) {
+	new Promise(res => {
```

Comment:
Alternatively you can use `async function` which always return a promise implicitly.